### PR TITLE
3 0 stable

### PR DESCRIPTION
**Pull request by jspizziri**
The specs still aren't passing, as they currently aren't in master. However, I upgraded the factory_girl gem, which is passing alot of the tests that are currently failing in master. In addition, upgraded rspec to v3.0.

I don't have the time to fix the specs right now, please feel free to fix them yourself if you merge. I'll try to come back to it later, but I can't guarantee when I'll have the space.

### CODE REVIEW

**Comment by halo** [comment]
Thank you very much for your work. @JDutil Would you mind having a quick look?

File: app/controllers/spree/digitals_controller.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module Spree
   class DigitalsController < Spree::StoreController
-    ssl_required :show
+    force_ssl only: :show
```
Comment (JDutil) [inline app/controllers/spree/digitals_controller.rb]: You probably want this to be conditional based on production env.

**Comment by JDutil** [comment]
I fixed up the 2-4-stable specs and committed the fixes to master as well.  Please rebase your work with master, and if there are still failing specs then there is still something you need to fix for Spree 3 compatibility.

**Comment by jspizziri** [comment]
I've rebased, this did fix a lot of the tests. However the master build is still failing. As for my most recent build, its still failing as a result of the spec returning 301 and a record not found error. I'm new to rspec but I think this has something to do with the test environment. Perhaps I didn't upgrade appropriately.

Any ideas?

**Comment by halo** [comment]
@jspizziri Thank you. I fixed that 301 at least, if you merge [this](https://github.com/jspizziri/spree_digital/pull/1/files) in. There was one other error for me though, let's see if it only happened locally for me...

**Comment by halo** [comment]
@jspizziri Oh, and do you by any chance feel like updating the README to indicate which spree version master is compatible with? :) Or remove that line, better no information than outdated information about  some`1.2.x` compatibility ;)

Also, not sure whether the [Versionfile](https://github.com/halo/spree_digital/blob/master/Versionfile) must be updated. If I understand you correctly, the `spree_digital` master is currently going for spree `2.3.x` and will be against `3.0.x` after merging in this pull request. Right?

**Comment by jspizziri** [comment]
@halo That is correct, I've bumped the versionfile and updated the language in the readme. Thanks for cleaning up those specs!

**Comment by halo** [comment]
Looks sweet to me, specs are [green](https://travis-ci.org/halo/spree_digital/builds/57646908). @JDutil If you would do me the honor and push the big button after one last look.

**Comment by jspizziri** [comment]
Not to be a pest, but it would be super sweet to get this merged.

**Comment by halo** [comment]
You are so far from being a pest ;)

**Comment by jspizziri** [comment]
Right on man. Would it also be good to create a 3-0-stable branch?

**Comment by halo** [comment]
Good idea. [Here](https://github.com/halo/spree_digital/tree/3-0-stable) it is.

Thank you indeed for your work.

**Comment by jspizziri** [comment]
:+1:

**Pull request merged**